deploy/roles/eod/tasks/main.yaml

```diff
@@ -49,10 +49,27 @@
   tags:
     - eod
 
+- name: Create admin group
+  group:
+    name: "{{ restricted_group_name }}"
+    state: present
+
```

Comment:
It is, we don't call the common role in the research_eod host of the research deployment. 
```
- hosts: research_eod
  become: yes
  become_method: sudo
  roles:
    - eod
    - odbc
```

We can either add the common role to that host, or create the group name like I did here.